packages/fast-components-react-base/src/viewport-positioner/viewport-positioner.tsx

```diff
@@ -364,6 +357,8 @@ class ViewportPositioner extends Foundation<
             if (this.state.validRefChecksRemaining > 0) {
                 this.setState({
                     validRefChecksRemaining: this.state.validRefChecksRemaining - 1,
+                    initialLayoutComplete:
+                        this.state.validRefChecksRemaining > 1 ? false : true,
```

Comment:
for true/false you can just do `initialLayoutComplete: this.state.validRefChecksRemaining <= 1` no need for ternary.